audio-player.js

```diff
@@ -57,6 +57,10 @@ module.exports = class AudioPlayer {
     this.removeEventListener = function(name, handler) {
       self.emitter.off(name, handler);
     };
+
+    this.fireEvent = function(name, args = []) {
+      self.emitter.emit(name, args);
```

Comment:
But then instead of changing code in one place I have to modify all fireEvent calls into emitter.emit. This is the lazier method and it's consistent with the others too